compiler/luci/import/include/luci/Import/CircleReader.h

```diff
@@ -152,6 +150,8 @@ class CircleReader
   circle::DataFormat native_data_format() const { return _native_subgraph->data_format(); }
   CircleMetadataSet native_metadata() const { return wrap(_native_model->metadata()); }
 
+  uint32_t num_subgraph() const { return wrap(_native_model->subgraphs()).size(); }
+
```

Comment:
Do we have to move the line?

Reply:
Yes, I move this methid definition below to other direct API methods definitions

Reply:
Before:

```cpp
public: // unpack API
  const CircleOperatorCodes_t &opcodes() const { return _model->operator_codes; }
  const CircleBuffers_t &buffers() const { return _model->buffers; }
  const CircleTensors_t &tensors() const { return _current_subgraph->tensors; }
  const CircleOperators_t &operators() const { return _current_subgraph->operators; }
  const std::vector<int32_t> &inputs() const { return _current_subgraph->inputs; }
  const std::vector<int32_t> &outputs() const { return _current_subgraph->outputs; }
  const std::string &name() const { return _current_subgraph->name; }
  const circle::DataFormat &data_format() const { return _current_subgraph->data_format; }
  const CircleMetadata_t &metadata() const { return _model->metadata; }

  const CircleTensorsPtr_t *tensors_ptr() const { return _tensors_ptr; }

  uint32_t num_subgraph() const { return _model->subgraphs.size(); }

  circle::BuiltinOperator builtin_code(const circle::OperatorT &op) const;
  std::string opcode_name(const circle::OperatorT &op) const;

public: // direct API
  CircleOperatorCodes native_opcodes() const { return wrap(_native_model->operator_codes()); }
  CircleBuffers native_buffers() const { return wrap(_native_model->buffers()); }
  CircleTensors native_tensors() const { return wrap(_native_subgraph->tensors()); }
  CircleOperators native_operators() const { return wrap(_native_subgraph->operators()); }
  VectorWrapper<int32_t> native_inputs() const { return wrap(_native_subgraph->inputs()); }
  VectorWrapper<int32_t> native_outputs() const { return wrap(_native_subgraph->outputs()); }
  std::string native_name() const { return fb_string2std_string(_native_subgraph->name()); }
  circle::DataFormat native_data_format() const { return _native_subgraph->data_format(); }
  CircleMetadataSet native_metadata() const { return wrap(_native_model->metadata()); }

  circle::BuiltinOperator builtin_code(const circle::Operator *op) const;
  std::string opcode_name(const circle::Operator *op) const;
```
After:

```cpp
public: // unpack API
  const CircleOperatorCodes_t &opcodes() const { return _model->operator_codes; }
  const CircleBuffers_t &buffers() const { return _model->buffers; }
  const CircleTensors_t &tensors() const { return _current_subgraph->tensors; }
  const CircleOperators_t &operators() const { return _current_subgraph->operators; }
  const std::vector<int32_t> &inputs() const { return _current_subgraph->inputs; }
  const std::vector<int32_t> &outputs() const { return _current_subgraph->outputs; }
  const std::string &name() const { return _current_subgraph->name; }
  const circle::DataFormat &data_format() const { return _current_subgraph->data_format; }
  const CircleMetadata_t &metadata() const { return _model->metadata; }

  const CircleTensorsPtr_t *tensors_ptr() const { return _tensors_ptr; }

  circle::BuiltinOperator builtin_code(const circle::OperatorT &op) const;
  std::string opcode_name(const circle::OperatorT &op) const;

public: // direct API
  CircleOperatorCodes native_opcodes() const { return wrap(_native_model->operator_codes()); }
  CircleBuffers native_buffers() const { return wrap(_native_model->buffers()); }
  CircleTensors native_tensors() const { return wrap(_native_subgraph->tensors()); }
  CircleOperators native_operators() const { return wrap(_native_subgraph->operators()); }
  VectorWrapper<int32_t> native_inputs() const { return wrap(_native_subgraph->inputs()); }
  VectorWrapper<int32_t> native_outputs() const { return wrap(_native_subgraph->outputs()); }
  std::string native_name() const { return fb_string2std_string(_native_subgraph->name()); }
  circle::DataFormat native_data_format() const { return _native_subgraph->data_format(); }
  CircleMetadataSet native_metadata() const { return wrap(_native_model->metadata()); }

  uint32_t num_subgraph() const { return wrap(_native_model->subgraphs()).size(); }

  circle::BuiltinOperator builtin_code(const circle::Operator *op) const;
  std::string opcode_name(const circle::Operator *op) const;
```